coreneuron/nrniv/main1.cpp

```diff
@@ -261,43 +265,43 @@ void nrn_init_and_load_data(int argc,
     }
 
     // pass by flag so existing tests do not need a changed nrn_setup prototype.
-    nrn_setup_multiple = nrnopt_get_int("--multiple");
-    nrn_setup_extracon = nrnopt_get_int("--extracon");
+    nrn_setup_multiple = cn_par.multiple;
+    nrn_setup_extracon = cn_par.extracon;
     // multisend options
-    use_multisend_ = nrnopt_get_flag("--multisend") ? 1 : 0;
-    n_multisend_interval = nrnopt_get_int("--ms-subintervals");
-    use_phase2_ = (nrnopt_get_int("--ms-phases") == 2) ? 1 : 0;
+    use_multisend_ = cn_par.multisend ? 1 : 0;
+    n_multisend_interval = cn_par.multisend;
+    use_phase2_ = (cn_par.ms_phases == 2) ? 1 : 0;
 
     // reading *.dat files and setting up the data structures, setting mindelay
     nrn_setup(filesdat.c_str(), is_mapping_needed, nrn_need_byteswap, run_setup_cleanup);
 
     // Allgather spike compression and  bin queuing.
-    nrn_use_bin_queue_ = nrnopt_get_flag("--binqueue");
-    int spkcompress = nrnopt_get_int("--spkcompress");
+    nrn_use_bin_queue_ = cn_par.binqueue;
+    int spkcompress = cn_par.spkcompress;
     nrnmpi_spike_compress(spkcompress, (spkcompress ? true : false), use_multisend_);
 
     report_mem_usage("After nrn_setup ");
 
     // Invoke PatternStim
-    if (nrnopt_get_flag("--pattern")) {
-        nrn_mkPatternStim(nrnopt_get_str("--pattern").c_str());
+    if (!cn_par.patternstim.empty()) {
+        nrn_mkPatternStim(cn_par.patternstim.c_str());
     }
 
     /// Setting the timeout
     nrn_set_timeout(200.);
 
     // show all configuration parameters for current run
-    nrnopt_show();
+    //nrnopt_show();
```

Comment:
The commented line can be deleted if it is not used any more

Reply:
My bad!